dashboard/db/migrate/20230515203259_create_parental_permission_requests.rb

```diff
@@ -0,0 +1,11 @@
+class CreateParentalPermissionRequests < ActiveRecord::Migration[6.1]
+  def change
+    create_table :parental_permission_requests do |t|
+      t.integer :user_id, null: false
```

Comment:
Is this the `user_id` from the User model? If so, I think you'll need to declare [an association](https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association) or foreign key here.